pkg/cli/alpha.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
I don't think that we should be updating the copyright

Reply:
Reverted.